MVC/application/model/userAccount.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+  class userAccount extends database
+  {
+    public $emailErrorMsg = false;
+    public $pwdErrorMsg = false;
+    public $duplicateEmailMsg = false;
+    public function loginRequest($userEmail, $userPassword){
+      $this->sendRequest("SELECT * FROM Account WHERE UserEmail = '$userEmail'");
+      if ($this->result->num_rows > 0) {
+        $row = $this->result->fetch_assoc();
```

Comment:
Let's try to add some details about the functionality of this code
Please explain the functionality of `fetch_assoc()` functions

Reply:
`fetch_assoc()` fetch the values from `$result` and return the associative array, like array key will be variable name and array value will be values. (Here : variable and values belongs to the database table)